packages/zeplin-extension-style-kit/values/gradient/linearColorStop.js

```diff
@@ -0,0 +1,44 @@
+import Color from "../color";
+import Percent from "../percent";
+
+class LinearColorStop {
+    /**
+     * LinearColorStop constructor
+     *
+     * @param {object} colorStopObject Color stop object, with color and relative position of the color stop in the range of [pFirst, pLast]
+     * @param {number} pFirst First color stop position
```

Comment:
This JSDoc definition seems to be outdated 😅

Reply:
🤫